src/frontends/jax/src/op/slice.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/slice.hpp"
+
+#include <vector>
+
+#include "openvino/core/node.hpp"
+#include "openvino/core/node_output.hpp"
+#include "openvino/frontend/jax/node_context.hpp"
+#include "openvino/op/constant.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace jax {
+namespace op {
+
+using namespace ov::op;
+
+OutputVector translate_slice(const NodeContext& context) {
+    num_inputs_check(context, 1, 1);
+    Output<Node> input = context.get_input(0);
+    auto start_indices_vector = context.const_named_param<std::vector<int64_t>>("start_indices");
+    auto limit_indices_vector = context.const_named_param<std::vector<int64_t>>("limit_indices");
+
+    auto start_indices =
+        std::make_shared<op::v0::Constant>(element::i64, Shape{start_indices_vector.size()}, start_indices_vector);
```

Comment:
This named param was created as a Constant in decoder, you converted it to vector of ints and created constant again, could you simplify this?